tests/fixtures/entities.py

```diff
@@ -2,22 +2,22 @@
 
 from openfisca_core.entities import Entity, GroupEntity
 
-from .variables import TestVariable
+from .variables import MyVariable
 
 
-class TestEntity(Entity):
-    def get_variable(self, variable_name):
-        result = TestVariable(self)
+class MyEntity(Entity):
```

Comment:
I understand we have a naming conflict with the `pytest` API. However, these new names are not satisfactory.
What about `Stub` or `Fixture`? 🙂